src/tests/stdlib/extract.ab

```diff
@@ -0,0 +1,15 @@
+import * from "std/fs"
+import { includes } from "std/array"
+main {
+    let tmpdir = trust $ mktemp -d /tmp/amber-XXXX $
+    cd tmpdir
+    trust $touch test.txt$
+    silent trust $tar -czf "filename.tar.gz" "{tmpdir}/test.txt"$
+    trust $rm "test.txt"$
+    let package = tmpdir + "/" + "filename.tar.gz"
+
+    if (extract(package, tmpdir)) {
```

Comment:
Would it also be worth verifying that the contents of the tar file are actually extracted?